### ISSUE TITLE
EDITOR variable doesn't work with paths that have spaces.

### ISSUE THREAD

**Issue by baileyn** [hugows/hf#4]
Setting the EDITOR environmental variable to something along the lines of `C:\Program Files\Sublime Text 3\subl.exe` doesn't work. I think it would work correctly with a path that doesn't contain spaces. 

Also, I believe that it would be better to put the default editor for windows as `notepad.exe` Since it will always be available from the command line, but `subl.exe` would only be available if it's in the PATH.

**Comment by hugows** [comment]
Right, maybe I could have subl the default only when it exists (Sublime Text 3 finally introduced this command so we don't have to create hacky subl.bat). 

I have to understand spaces on windwos better, as I'm having several problems with it. Thanks for opening the issues.
